libcxx/test/std/time/time.zone/time.zone.db/time.zone.db.tzdb/tzdb.members.pass.cpp

```diff
@@ -37,7 +37,14 @@ int main(int, const char**) {
   tzdb.version = "version";
   assert(tzdb.version == "version");
 
-  // TODO TZDB add the other data members
+  // TODO TZDB update the tests above to use this style.
+  { [[maybe_unused]] std::same_as<std::string> auto _ = tzdb.version = "version"; }
+  { [[maybe_unused]] std::same_as<std::vector<std::chrono::time_zone>> auto& _ = tzdb.zones; }
+  tzdb.zones.clear(); // is it a non-const member?
```

Comment:
I would suggest doing `std::vector<std::chrono::time_zone>& zones = tzdb.zones;` -- that ensures the right type and looks a bit more straightforward to me.